findIndexImage/main.go

```diff
@@ -17,14 +17,16 @@ func main() {
 	nhc := "red-hat-workload-availability-node-healthcheck-operator-bundle:v"
 	snr := "red-hat-workload-availability-self-node-remediation-bundle:v"
 	nmo := "red-hat-workload-availability-node-maintenance-operator-bundle:v"
+	mdr := "red-hat-workload-availability-machine-deletion-remediation-operator-bundle:v"
+	far := "red-hat-workload-availability-fence-agents-remediation-bundle:v"
```

Comment:
Have you checked it already?
I think it is supposed to be `fence-agents-remediation-operator-bundle`

Reply:
Yes, I checked, with `fence-agents-remediation-operator-bundle` did not work
```
red-hat-workload-availability-fence-agents-remediation-operator-bundle:v not found, last build too old?!
```

with `fence-agents-remediation-bundle` instead

```
OCP v4.11                                                                                                                                                                                                                                    
  registry-proxy.engineering.redhat.com/rh-osbs/red-hat-workload-availability-fence-agents-remediation-bundle:v0.1.0-18                                                                                                                      
  registry-proxy.engineering.redhat.com/rh-osbs/iib:459248                                                                                                                                                                                   
  Wed, 22 Mar 2023 18:29:32 UTC                                                                                                                                                                                                              
                                                                                                                                                                                                                                             
OCP v4.12                                                                                                                                                                                                                                    
  registry-proxy.engineering.redhat.com/rh-osbs/red-hat-workload-availability-fence-agents-remediation-bundle:v0.1.0-18                                                                                                                      
  registry-proxy.engineering.redhat.com/rh-osbs/iib:459259                                                                                                                                                                                   
  Wed, 22 Mar 2023 18:41:12 UTC                                                                                                                                                                                                              
                                                                                                                                                                                                                                             
OCP v4.13                                                                                                                                                                                                                                    
  registry-proxy.engineering.redhat.com/rh-osbs/red-hat-workload-availability-fence-agents-remediation-bundle:v0.1.0-18                                                                                                                      
  registry-proxy.engineering.redhat.com/rh-osbs/iib:459273                                                                                                                                                                                   
  Wed, 22 Mar 2023 18:47:19 UT
```

Reply:
Apparently I have used the word `operator` for any reference of the bundle container except it's name in midstream Dockerfile...